### PR TITLE
docs(guides): add 'Query Development Tutorial' section in creating-queries.md

### PR DESCRIPTION
**Pull request by rafaela-soares**
Signed-off-by: Rafaela Soares <rafaela.soares@checkmarx.com>

**Proposed Changes**
- Added 'Query Development Tutorial' section in creating-queries.md

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
This is something that I would like to get input from @kaplanlior

File: docs/creating-queries.md [hunk 1]
```diff
@@ -10,12 +10,191 @@ So for example, if we wanted to transform a .tf file in ./code/test we could typ
...
+  "descriptionText": "Check if MultiRegion is Enabled",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudtrail#is_multi_region_trail",
+  "platform": "Terraform"
+}
+
+```
+
+👨‍💻 **Test Folder**
+
+Keep in mind that all the positive and negative files should contain only one breaking point case. This way, the results are more clear. As a best practice, the test folder should contain all the [extensions available](https://docs.kics.io/1.3.3/platforms/) by the platform.
```
Comment (rogeriopeixotocx) [inline docs/creating-queries.md:143]:
```suggestion
Keep in mind that all the positive and negative files should contain only one breaking point case. This way, the results are more clear. As a best practice, the test folder should contain all the [extensions available](https://docs.kics.io/latest/platforms/) by the platform.
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]

File: docs/creating-queries.md [hunk 2]
```diff
@@ -10,12 +10,191 @@ So for example, if we wanted to transform a .tf file in ./code/test we could typ
...
+
+To test the query in the [Rego playground](https://play.openpolicyagent.org/), perform the following steps:
+1. Enable Coverage (highlights which statements were evaluated);
+2. In the input, paste the payload (explained above);
+3. In the playground, paste and/or write the policies. Check if the statements are painted green;
+4. Click on Evaluate;
+5. In the output, check if the results are as expected.
+
+
+<img alt="RegoPlayground" src="img/rego_playground_example_tf.PNG">
```
Comment (felipe-avelar) [inline docs/creating-queries.md:121]: I think this image source link is wrong

File: docs/creating-queries.md [hunk 3]
```diff
@@ -52,3 +231,118 @@ go run ./cmd/console/main.go generate-id
...
+      "id": "c7000383-16d0-4509-8cd3-585e5ea2e2f2",
+      "queryName": "Invalid Contact URL (v2)",
+      "descriptionUrl": "https://swagger.io/specification/v2/#contactObject"
+    }
+  },
+  "aggregation": 2
+}
+
+```
+
```
Comment (felipe-avelar) [inline docs/creating-queries.md:273]: I think it is missing `override` keyword

**Review by felipe-avelar — CHANGES_REQUESTED** [review]
Pleas, check comments

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**